### DISCUSSION TITLE
Add sort feature based on date created or upvotes etc

### DISCUSSION THREAD

**Discussion by shanisnake** [dailydotdev/daily#1444] [Feature Requests]
I'd like to request a feature that allows users to sort content on Daily.dev based on different criteria, such as:

Date Created: Sort articles by their publication date, with options for newest or oldest first.
Upvotes/Popularity: Sort articles by the number of upvotes or interactions, so the most popular content appears at the top.

Custom Filters: 
Allow users to combine or choose additional filters, such as trending articles, most commented, etc.
This feature would enhance content discovery and allow users to prioritise articles based on their preferences.

**Comment by idoshamun** [comment]
Hi @shanisnake,

Have you tried our [explore](https://app.daily.dev/posts) page?

**Reply by KeshiaRose** [reply]
Not OP but I just want to be able to sort my feed and my custom feeds by date. New content is being buried.

**Reply by idoshamun** [reply]
yap that's on the backlog but no release date yet unfortunately
